src/components/Layout/PhoneNumberInput.tsx

```diff
@@ -63,6 +64,14 @@ export const PhoneNumberInput: FunctionComponent<{
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   onSubmit = () => {}
 }) => {
+  const phoneUtil = PhoneNumberUtil.getInstance();
+  const value: string =
+    mobileNumberValue.length > 2
+      ? phoneUtil.format(
+          phoneUtil.parse(mobileNumberValue, "SG"),
+          PhoneNumberFormat.NATIONAL
+        )
+      : mobileNumberValue;
```

Comment:
Tested this out, it threw an error when I entered a really long number, please look into it!

The error thrown was: `Error: The string supplied is too long to be a phone number`.
Have also sent you a screen recording of it!

Reply:
fix by restricting input to 8 digits.

Reply:
@shermainelim
fix:phone number input using constant value that is found on google libphonenumber library

Reference: https://github.com/ruimarinho/google-libphonenumber/blob/master/src/phonenumberutil.js

Line 114

onChange we getting the max length of phone number from MAX_LENGTH_FOR_NSN then
then is validated. if wrong, then don't update the state.

Reply:
While the value for `MAX_LENGTH_FOR_NSN` is 17 for now, it may not be 17 in the future.

Have you considered leveraging the errors that `format` and `parse` will throw when met with an input that is too short or too long?

Reply:
yes, instead of 17 , have replaced with phoneNumberFormatted function that leverages on `format` and `parse`
for different kinds of inputs.